internal/config/config.go

```diff
@@ -89,8 +89,6 @@ func read() Config {
 		return applyEnvironmentConfiguration(config)
 	}
 
-	fmt.Printf("Testcontainers properties file has been found: %s\n", tcProp)
```

Comment:
I noticed this print was here affecting the Go Examples, which matches the output of a test. That's why I'm removing it here